apps/web/app/hooks/features/useTimeLogs.ts

```diff
@@ -43,7 +43,6 @@ export function useTimeLogs() {
 			profile.member?.employeeId,
 			queryTimerLogsDailyReport,
 			setTimerLogsDailyReport,
-			user?.employee.id,
```

Comment:
@GedeonTS this looks strange that you removing field?? Are you sure it was not required to pass it? What was warning here that you try to fix!?

Reply:
Hi sir @evereq , I was trying to fix the following warning `Warning: React Hook useEffect has an unnecessary dependency: 'triggerRef.current'. Either exclude it or remove the dependency array.`